overlord/snapstate/catalogrefresh.go

```diff
@@ -71,7 +79,13 @@ func (r *catalogRefresh) Ensure() error {
 		return nil
 	}
 
-	next := now.Add(catalogRefreshDelay)
+	delay := catalogRefreshDelayBase
+	if !catalogExists {
+		// if the catalog didn't exist, add the delta for the next refresh
+		// (otherwise the delta has already been added)
+		delay = catalogRefreshDelayWithDelta
```

Comment:
This is 24h + rand(6h) in the future - am I misreading here? If not is that the delay we want if we don't have a catalog? Or it it rather just rand(6h)?

Reply:
we will try now (that check is done before this) and then yes wait that delay (24h+) for the next time, but the old code was the same, it would wait 24h independently of success or not

Reply:
24h + rand(6h) is what we want (this is the "average of 27h" we talked about in the standup yesterday)

Reply:
Thanks